README.md

```diff
@@ -333,6 +333,38 @@ If *rotation* is specified, sets the projection’s [three-axis rotation](http:/
 
 If *precision* is specified, sets the threshold for the projection’s [adaptive resampling](http://bl.ocks.org/mbostock/3795544) to the specified value in pixels and returns the projection. This value corresponds to the [Douglas–Peucker](http://en.wikipedia.org/wiki/Ramer–Douglas–Peucker_algorithm) distance. If *precision* is not specified, returns the projection’s current resampling precision which defaults to √0.5 ≅ 0.70710…
 
+<a href="#projection_fitSize" name="projection_fitSize">#</a> <i>projection</i>.<b>fitSize</b>(<i>size</i>, <i>object</i>)
+
+Updates the projection's [scale](#projection_scale) and [translate](#projection_translate) to fit the specified GeoJSON *object* in the center of the bounding box defined by *size*. The *size* is specified as a two-element array [*width*, *height*] (typically in pixels). Any [clipExtent](#projection_clipExtent) is ignored when determining the new scale and translate. Returns the projection.
+
+For example, this would scale and translate the [New Jersey State Plane projection](http://bl.ocks.org/mbostock/5126418) to fit the GeoJSON object *nj* in the center of a 960x500 bounding box:
```

Comment:
Obsessive style nits:

* Use curved quote (“projection’s” instead of “projection's”).
* Use “clip extent” when writing plain English (as here; use *projection*.clipExtent when referring specifically to the method).
* Use “960×500” instead of “960x500”.